src/test/kotlin/step2/StringCalculatorTest.kt

```diff
@@ -0,0 +1,106 @@
+package step2
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+
+class StringCalculatorTest {
+
+    @ParameterizedTest
+    @ValueSource(
+        strings = [
+            "2 + 3 * 4 / 2 = 10",
+            "2 = 2",
+            "2 + 5 = 7"
+        ]
+    )
+    fun `계산 성공`(input: String) {
+        input.split(" = ").also {
+            val result = StringCalculator.execute(it.first())
+            assertThat(result).isEqualTo(it.last().toInt())
+        }
+    }
+
+    @ParameterizedTest
+    @ValueSource(
+        strings = [
+            "2 + 3 = 5",
+            "2 + 7 = 9",
+            "2 + 0 + 9 + 3 = 14"
+        ]
+    )
+    fun `덧셈 테스트`(input: String) {
+        input.split(" = ").also {
+            val result = Calculator.calculate(it.first())
+            assertThat(result).isEqualTo(it.last().toInt())
+        }
+    }
+
+    @ParameterizedTest
+    @ValueSource(
+        strings = [
+            "5 - 3 = 2",
+            "7 - 2 = 5",
+            "2 - 1 = 1"
+        ]
+    )
+    fun `뺄셈 테스트`(input: String) {
+        input.split(" = ").also {
+            val result = Calculator.calculate(it.first())
+            assertThat(result).isEqualTo(it.last().toInt())
+        }
+    }
+
+    @ParameterizedTest
+    @ValueSource(
+        strings = [
+            "6 / 3 = 2",
+            "8 / 2 = 4",
+            "9 / 1 = 9"
+        ]
+    )
+    fun `나눗셈 테스트`(input: String) {
+        input.split(" = ").also {
+            val result = Calculator.calculate(it.first())
+            assertThat(result).isEqualTo(it.last().toInt())
+        }
+    }
+
+    @ParameterizedTest
+    @ValueSource(
+        strings = [
+            "6 * 8 = 48",
+            "8 * 2 = 16",
+            "9 * 9 = 81"
+        ]
+    )
+    fun `곱셈 테스트`(input: String) {
+        input.split(" = ").also {
+            val result = Calculator.calculate(it.first())
+            assertThat(result).isEqualTo(it.last().toInt())
+        }
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = ["", " "])
+    fun `예외 발생 - input값이 null 또는 empty`(input: String?) {
+        assertThatThrownBy {
+            InputValidator.validate(input)
+        }.isInstanceOf(IllegalArgumentException::class.java)
```

Comment:
```suggestion
        assertThrows<IllegalArgumentException> { InputValidator.validate(input) }
```
`aseertThatThrownBy` 를 사용하면 `Assertions.java` 를 사용하게 됩니다. `Assertions`를 `Kotlin`에 맞게 구현 된 `assertThrows<>`를 사용해보시는건 어떨까요? 🤔

Reply:
수정 반영했습니다~